features/comments_and_kudos/admin_info.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Some admins can see IP addresses and emails for comments
+  Scenario: Admin info for comments isn't visible to the work's owner
```

Comment:
Work owner covers logged in users, but can we also test that admin info isn't visible to a guest?